example-apps/typescript/src/test/creates.test.ts

```diff
@@ -16,4 +16,45 @@ describe('movie', () => {
       year: 2020,
     });
   });
+
+  test('create movies in bulk', async () => {
+    const genre = 'Genre Test';
+    const idem1 = 'test-id-xxx';
+    const idem2 = 'test-id-yyy';
+    const idem3 = 'test-id-zzz';
+
+    const bulkBundle = {
+      bulk: [
+        {
+          inputData: { genre, title: 'Title 1', year: 2020 },
+          meta: { id: idem1 },
+        },
+        {
+          inputData: { genre, title: 'Title 2', year: 2021 },
+          meta: { id: idem2 },
+        },
+        {
+          inputData: { genre, title: 'Title 3', year: 2022 },
+          meta: { id: idem3 },
+        },
+      ],
+      groupedBy: { genre },
+    };
+
+    const result = await appTester(App.creates.movies.operation.performBulk, bulkBundle);
+    expect(result).toMatchObject({
+      idem1: {
+        outputData: { genre, id: '1', title: 'Title 1', year: 2020 },
+        error: null,
+      },
+      idem2: {
+        outputData: { genre, id: '2', title: 'Title 2', year: 2021 },
+        error: null,
+      },
+      idem3: {
+        outputData: { genre, id: '3', title: 'Title 3', year: 2022 },
+        error: null,
+      },
+    });
+  });
```

Comment:
@eliangcs Is this the test you referred to in the ticket or a manual one?

Reply:
Also, the `Bundle` and the new `BulkBundle` are mixed up in the [failing tests](https://github.com/zapier/zapier-platform/actions/runs/9086671182/job/24972860812?pr=786) in this file. I'm confused about this.

Reply:
@kola-er sorry for the confusion! A manual test should be enough: Manually create a TypeScript integration project, add a create with `performBulk`, and use the new `index.d.ts` to see it's working as expected.

Reply:
Thank you.